homeassistant/const.py

```diff
@@ -228,6 +228,10 @@
 VOLUME_GALLONS = 'gal'  # type: str
 VOLUME_FLUID_OUNCE = 'fl. oz.'  # type: str
 
+# Bandwidth units
+BANDWIDTH_MEGABITS_SECONDS = 'Mbps'  # type: str
+BANDWIDTH_GIGABITS_SECONDS = 'Gbps'  # type: str
```

Comment:
So why there is not unit of this kind in the const.py ? 
Anyway Mb/s and MB/s are not the same unit. The small b means bit, the other (B) means byte (https://www.softperfect.com/contact/knowledgebase.php?article=10)